src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/add.svg');
+    background-position: setting-get('rhythm') 49%;
```

Comment:
Why is this `49%`? Why not `50%`? If it's because it doesn't vertically or horizontally center properly, use `calc` to nudge it in to the right position